lib/perl/Genome/VariantReporting/Framework/Component/Expert/Command.pm

```diff
@@ -63,7 +145,7 @@ sub is_many_input_names {
     my $self = shift;
 
     my @properties = $self->__meta__->properties(
-        is_many => 1, is_input => 1);
+        is_many => 1, is_argument => 1);
```

Comment:
maybe we could eliminate the need for the is_argument specification by looking at the properties on the result class instead and only getting those?